CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.0.0] - 2021-08-02
```

Comment:
Is this a version 3.0.0? We didn't break any functionality, we just added Pro-E support so it feels more like a v2.1.0 to me?

Reply:
You can not upgraded parakeet-ros versions without bumping parakeet-sdk versions.  In my mind, that is a breaking change as it requires the user to do extra work to update versions. What is your take on it?